hv-rhel7.x/hv/Makefile

```diff
@@ -10,16 +10,20 @@ obj-m	+= hyperv-keyboard.o
 # Get machine bit length
 arch=$(shell getconf LONG_BIT)
 
-# RDMA should only build on 64 bits arch
-ifeq ($(arch), 64)
-        obj-m += hv_network_direct.o
-endif
-
 # Get RHEL/CentOS version information
 rhel_major=$(shell grep -Eoh [0-9]+\.[0-9]+ /etc/{issue,*release} | head -1 | awk -F'.' '{ print $$1 }')
 rhel_minor=$(shell grep -Eoh [0-9]+\.[0-9]+ /etc/{issue,*release} | head -1 | awk -F'.' '{ print $$2 }')
 rhel_release_code=$(shell echo $$(($(rhel_major) << 8 | $(rhel_minor))))
 
+# RDMA should only build on 64 bits arch
+ifeq ($(arch), 64)
+# Network direct driver in LIS is supported for RHEL/CentOS <= 7.6
+build_hv_network_direct=$(shell test $(rhel_release_code) -lt 1799; echo $$?)
```

Comment:
Use tab space for inner lines of condition.

Reply:
Done.